.github/workflows/auto-pr-merge.yml

```
@@ -106,13 +107,15 @@ jobs:
                 const twitter_logo = "https://edent.github.io/SuperTinyIcons/images/svg/twitter.svg"
                 const fb_logo = "https://edent.github.io/SuperTinyIcons/images/svg/facebook.svg"
                 const linkedin_logo = "https://edent.github.io/SuperTinyIcons/images/svg/linkedin.svg";
+                const dev_logo = "https://d2fltix0v2e0sb.cloudfront.net/dev-badge.svg";
```

```suggestion
                const dev_logo = "https://edent.github.io/SuperTinyIcons/images/svg/dev_to.svg";
```

This one is smaller around 600 bytes

---

changed the URL to suggested one.